core/embed/prodtest/main.c

```diff
@@ -65,8 +65,11 @@
 
 #include "memzero.h"
 
-#ifdef STM32U5
+#ifdef USE_STORAGE_HWKEY
 #include "secure_aes.h"
+#endif
+
+#ifdef STM32U5
```

Comment:
residual STM32U5 define, needed for the CPU ID function, but we should get rid of it at some point, lets make an issue

Reply:
I agree and created #4306